perl/lib/Gherkin/Exceptions.pm

```diff
@@ -130,11 +132,10 @@ sub new {
 
 sub detailed_message {
     my $self = shift;
-    return sprintf(
-        "expected: %s, got '%s'",
+    return sprintf
+        'expected: %s, got \'%s\'',
```

Comment:
Please change to:

```suggestion
        q{expected: %s, got '%s'},
```

I don't think having the escaped single quotes makes the code better than having double quotes; the suggestion above removes the need to escape the single quotes at the expense of making the code more idiomatic Perl, but less "simplified Perl".